hazelcast-spring/src/test/java/com/hazelcast/spring/DummySSLContextFactory.java

```diff
@@ -16,18 +16,17 @@
 
```

Comment:
Do we need to keep this class?

Reply:
Yes. It's used in Spring context testing. We have it fully tested on the OS side.